### DISCUSSION TITLE
In over my head

### DISCUSSION THREAD

**Discussion by KathCobb** [community/community#121832] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm sure I am in the wrong place but I really need some help. I am so far in over my head.  I have a Late 2015 iMac and was looking into using it as a display with my MacBook Pro.  I stumbled across OpenCore patcher that stated it could enable airplay.  Great. except not great. I did not end up using the airplay connection and just went back to using my iMac as I always have with two external display's connected via thunderbolt ports. I use this iMac every day for work.  It's old but it's been a champ, I have been trying to figure out how to move away from it.  Anyway, after about 24 hours I lost both my external monitors on the iMac. I really do not know what to do to fix this.  I NEED those display's back.  Can someone guide me through how to get my iMac back to the way it was, or to get my displays back or even suggest to me where I could go for help?  I have absolutely no idea what to do from here. Any help at all would be appreciated.

**Comment by ebndev** [comment]
Hi @KathCobb,

Thanks for posting in the GitHub community!

You might want to try asking for help somewhere that may be more likely to be able to resolve your issue like Apple's Genius Bar or another forum like the Apple subreddit.

We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.
